assets/blocks/index.js

```diff
@@ -13,24 +13,46 @@ const attributes = {
 registerBlockVariation( 'core/button', {
 	name: 'sensei-certificates/view-certificate-button',
 	title: __( 'View Certificate', 'sensei-certificates' ),
-	description: __( 'Enable a learner to view their course certificate.', 'sensei-certificates' ),
-	keywords: [
-		__( 'Certificates', 'sensei-lms' ),
-	],
+	description: __(
+		'Enable students to view their course certificate.',
+		'sensei-certificates'
+	),
+	keywords: [ __( 'Certificates', 'sensei-lms' ) ],
 	category: 'sensei-lms',
 	attributes,
 	isActive: ( blockAttributes, variationAttributes ) =>
 		blockAttributes.className?.match( variationAttributes.className ),
 } );
 
-const addBlockToTemplate = ( blocks ) => ( [
+const addBlockToTemplate = ( blocks ) => [
 	...blocks,
-	[ 'core/button', attributes ]
-] );
+	[ 'core/button', attributes ],
+];
 
 // Add this block to the Course Completed Actions block.
 addFilter(
 	'sensei-lms.Course.completedActions',
 	'sensei-certificates',
 	addBlockToTemplate
 );
+
+registerBlockVariation( 'core/buttons', {
+	name: 'sensei-certificates/view-certificate-buttons',
+	title: __( 'View Certificate', 'sensei-certificates' ),
+	description: __(
+		'Enable students to view their course certificate.',
+		'sensei-certificates'
+	),
+	category: 'sensei-lms',
+	keywords: [
+		__( 'Certificate', 'sensei-certificates' ),
+		__( 'View Certificate', 'sensei-certificates' ),
+	],
+	innerBlocks: [ [ 'core/button', attributes ] ],
+	templateLock: 'all',
+	attributes: {
```

Comment:
Since it's just a "helper" to include the child button inside a buttons block, I think we could remove the `attributes` and the `isActive`, so after added the user will see it as the Buttons block having the "View Certificate" block. Does it make sense to you?